### PR TITLE
Fix comments in config file

### PR DESCRIPTION
**Pull request by FryDay**
Some of the defaults in the comments do not match the actual default values.

### CODE REVIEW

**Comment by MaxLeiter** [comment]
Looks good to me

**Comment by YaManicKill** [comment]
Yep, lgtm too.

File: defaults/config.js [hunk 1]
```diff
@@ -30,7 +30,7 @@ module.exports = {
 	// Set the local IP to bind to.
 	//
 	// @type     string
-	// @default  "0.0.0.0"
+	// @default  undefined
```
Comment (xPaw) [inline defaults/config.js]: Could this perhaps explain what undefined means in this context?

Comment (YaManicKill) [inline defaults/config.js]: Yeah, that's probably a good plan. Just a short explanation that undefined means it'll listen on everything.

**Comment by astorije** [comment]
Hey @FryDay, could you get rid of the merge commit in the history please? Let me know if you don't know how to do that and we can sync on IRC.

In the future, when you want to sync with master, always use `git rebase master` instead of merge or pull (pull is simply fetch + merge).

Thanks a lot for fixing cruft!!! :smile:

**Comment by FryDay** [comment]
@astorije certainly, I'll take care of it tonight when I get home.

**Comment by astorije** [comment]
Perfect :-)

**Comment by FryDay** [comment]
@astorije I believe I fixed it. If it doesn't look okay, let me know.

**Comment by astorije** [comment]
I'm :+1: on this, going for second review (as there were changes after @YaManicKill's comment).
@FryDay, maybe squash the commits into one if you happen to stop by before @YaManicKill gives his :+1:, otherwise not a huge deal.

**Comment by YaManicKill** [comment]
:+1: 

I'm fine with teh 2 commits because they are both doing logically seperate things. Merging.

**Pull request merged**